test/scenarios/el92-src@offline.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+guest_name=host1
```

Comment:
Nit: can we use uppercase since this is a global variable? 
i.e. OFFLINE_VM_NAME